lighthouse-core/audits/metrics/interactive.js

```diff
@@ -12,7 +12,7 @@ const UIStrings = {
   /** The name of the metric that marks the time at which the page is fully loaded and is able to quickly respond to user input (clicks, taps, and keypresses feel responsive). Shown to users as the label for the numeric metric value. Ideally fits within a ~40 character limit. */
   title: 'Time to Interactive',
   /** Description of the Time to Interactive (TTI) metric, which evaluates when a page has completed its primary network activity and main thread work. This is displayed within a tooltip when the user hovers on the metric name to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
-  description: 'Interactive marks the time at which the page is fully interactive. ' +
```

Comment:
We went with this because the actual moment is called `interactive` and the metric is the timespan until that moment *Time To* Interactive. Not opposed to rewording so that the name of the metric is the subject but we'll need to change up the rest of the sentence then :)

Reply:
Yes, this was my intention to bring the title to first words in the description as everywhere else already.

Reply:
Would it be better as "Time to interactive is the amount of time it takes for the page to become fully interactive?"  Since yea, there is a unit disconnect between a span of time (tti) and a point in time (interactive).

nit: What is the capitalization: `Time to Interactive` or `Time to interactive`?